packages/client/src/components/notebook/operations/DefaultOperation.tsx

```diff
@@ -24,6 +25,15 @@ export const DefaultOperation = observer(
             return <StyledJson>{output}</StyledJson>;
         }
 
+        if (typeof output === 'object') {
```

Comment:
Should be able to reference your utility method here too

Reply:
Utility method was added to check if the json string is json and replaced this statement.

Reply:
Resolved this, we check if output is both string and object and then use utility function to resolve.